src/interpreter/util/expression/expression-contains-super-keyword.ts

```diff
@@ -12,10 +12,9 @@ export function expressionContainsSuperKeyword(expression: TS.Expression | TS.Pr
 		return expressionContainsSuperKeyword(expression.expression, typescript) || expressionContainsSuperKeyword(expression.argumentExpression, typescript);
 	} else if (typescript.isParenthesizedExpression(expression)) return expressionContainsSuperKeyword(expression.expression, typescript);
 	else if (typescript.isAsExpression(expression)) return expressionContainsSuperKeyword(expression.expression, typescript);
-	else if (typescript.isTypeAssertionExpression?.(expression) || (!("isTypeAssertionExpression" in typescript) && (typescript as typeof TS).isTypeAssertion(expression))) {
-		return expressionContainsSuperKeyword(expression.expression, typescript);
-	}
-	else {
+	else if (typescript.isTypeAssertionExpression?.(expression) || (!("isTypeAssertionExpression" in typescript) && (typescript as any).isTypeAssertion(expression))) {
```

Comment:
Same feedback as above - we can write this like this instead:

```ts
else if (typescript.isTypeAssertionExpression?.(expression) || (!("isTypeAssertionExpression" in typescript) && (typescript as {isTypeAssertion: typeof TS.isTypeAssertionExpression}).isTypeAssertion(expression))) {
		return expressionContainsSuperKeyword(expression.expression, typescript);
}
```

That way we won't have to do the `expression as TS.TypeAssertion` cast